src/main/seal.c

```diff
@@ -269,9 +269,9 @@ int main(int argc, char **argv)
   // Some options don't do anything with -g, so warn about that now.
   if(boolTrialOnly)
   {
-    if(outPath != NULL || inPath != NULL || cipherString != NULL || forceOverwrite || expected_policy != NULL)
+    if(authString != NULL || ownerAuthPasswd != NULL || outPath != NULL || inPath != NULL || cipherString != NULL || forceOverwrite || expected_policy != NULL)
```

Comment:
When I tested, I got the warning message no matter what. In looking at it, I think it was because `ownerAuthPasswd` is initialized to the empty string (""), not NULL. When I changed the criteria to `ownerAuthPasswd != ""`, therefore, this corrected it.

```suggestion
    if(authString != NULL || ownerAuthPasswd != "" || outPath != NULL || inPath != NULL || cipherString != NULL || forceOverwrite || expected_policy != NULL)
```